src/containers/DataGUI.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
Ops. I think we cannot comment the complete file. Sorry for that.

Maybe you can pass some props to add those styles and for others things to differentiate.